### PR TITLE
Polyp Food Recipes and Livestock Crate

### PR DESCRIPTION
**Pull request by Optimism333**
## Floating, gurgling lifeforms have been spotted in Nanotrasen space... ##
polyp

I finally made an attempt to give spess polyps some ranching potential, expanding on their role as an ayy cattle animal. There's now a few cooking recipes that use their meat and/or gelatin. It's not very many, but it's a start.

Now that they have some ranching potential, they can be ordered from cargo for 75 credits each. I might up the price or make them minor contraband depending on feedback, since they're a little more dangerous than the average farm animal.

Lucky for amateur herders, shipments of polyps come with a useful set of instructions to avoid accidents.

polyp_instructions

## What the pr does and why ##
- Adds three simple dishes that use space polyp meat or their gelatin: polyp burger, polypwich, and polyp pudding.
- Adds space polyps to the cargo hydroponics livestock shipments. The crates come with a useful set of instructions for amateur herders.

polypfoods

## Conclusion ##
Please do share thoughts and feedback. If this passes, I will update the wiki appropriately.

:cl:
 * rscadd: Adds some cooking recipes that use space polyp meat and gelatin
 * rscadd: Space polyps can now be ordered as livestock from cargo, for cooking or other reasons

### CODE REVIEW

**Comment by ManopolisJones** [comment]
ah yes it's finally here, the funky jelly polyp with his jelly meals

**Comment by ValkyrieSkies** [comment]
What's dangerous about them?

**Comment by Optimism333** [comment]
> What's dangerous about them?

They're a bit tougher than a goat or cow, so harder to slaughter. And they can give a mean burning sting, so making a large group of them mad isn't a good idea.

**Comment by ValkyrieSkies** [comment]
> > What's dangerous about them?
> 
> They're a bit tougher than a goat or cow, so harder to slaughter. And they can give a mean burning sting, so making a large group of them mad isn't a good idea.

Do they anger easily?

**Comment by Optimism333** [comment]
> > > What's dangerous about them?
> > 
> > 
> > They're a bit tougher than a goat or cow, so harder to slaughter. And they can give a mean burning sting, so making a large group of them mad isn't a good idea.
> 
> Do they anger easily?

Less easily than goats, so they won't get mad for no reason. But hitting them or other actions that make retaliation mobs angry will do it.

**Comment by Eneocho** [comment]
Goats are more immediately dangerous and there are better ways to gunk through cargo, keeping it as a regular purchaseable seems ok.

**Comment by Optimism333** [comment]
Added a useful set of instructions that comes with the space polyp crate. Was also a good experience for learning how to write bureaucracy straight into the code.

Screenshots above have been updated.

File: code/modules/reagents/reagent_containers/food/snacks.dm [hunk 1]
```diff
@@ -2730,6 +2742,17 @@
 	reagents.add_reagent(NUTRIMENT, 7)
 	bitesize = 2
 
+/obj/item/weapon/reagent_containers/food/snacks/polypwich
+	name = "Polypwich"
+	desc = "Polyp meat and gelatin between two slices of bread makes for a nutritious sandwich. Unfortunately it has a soggy and unpleasant texture. These are commonly served to mothership prisoners who misbehave."
```
Comment (Eneocho) [inline code/modules/reagents/reagent_containers/food/snacks.dm:2747]: Could use a small chance to make you get a nausea message or something. "The texture almost makes you vomit!"

Comment (Optimism333) [reply code/modules/reagents/reagent_containers/food/snacks.dm:2747]: That could be fun. I'll look into it after I've had some sleep.

Comment (Optimism333) [reply code/modules/reagents/reagent_containers/food/snacks.dm:2747]: Added one.

**Comment by KingofHONK** [comment]
All you need is a new reagent or two and this is 10/10

Oh fuck even better what if they could adopt plant qualities like glowing and whatever chems are spliced into it are added to their stings? That'd be sick

**Comment by Optimism333** [comment]
> All you need is a new reagent or two and this is 10/10
> 
> Oh fuck even better what if they could adopt plant qualities like glowing and whatever chems are spliced into it are added to their stings? That'd be sick

What sort of new reagent where you thinking of, and did you have an idea for how it should be gotten? I don't think I can do the splicing thing, unfortunately. But maybe I can make another reagent and a way to acquire it.

**Comment by Eneocho** [comment]
You could check the beecode and the honey reagent system for inspiration on how to make the polyps get reagents from unharvested plants, a reagent check could enable mutations on the polyp, so a polyp that fed on uranium would enable it's glowing trait (ghoulcode).

**Comment by Optimism333** [comment]
> You could check the beecode and the honey reagent system for inspiration on how to make the polyps get reagents from unharvested plants, a reagent check could enable mutations on the polyp, so a polyp that fed on uranium would enable it's glowing trait (ghoulcode).

I could, but that's beyond the scope of what I'm willing to do for the time being. Maybe if I revisit these guys and ayy hydroponics in general once I'm done with my pr to-do list.

**Comment by Eneocho** [comment]
I'm just spitballing ideas here. This PR seems good enough as is and I can't wait for more service content.

**Pull request merged**